portal/src/texts/core/elevation.md

```diff
@@ -1,3 +1,17 @@
 ## Elevasjon
 
-Vi ønsker at brukerne lett skal se hvilket innhold som er på ett nivå og hvilket som er over- eller underordnet. Til det bruker vi blant annet skygger, som skaper inntrykk av elevasjon. Vi bruker ulike skygger for å skape et hierarki.
+Prinsippet elevasjon handler om å løfte det som er viktig og skape et intuitivt informasjonshierarki. Det kan blant annet gjøres ved å jobbe med nivåer og dybde.
+
+Bruk av skygge indikerer at elementer blir aktivert. Forskyvning ved scroll skaper også dybde og legger viktig innhold høyere opp og nærmere brukeren. Man kan også skape nivåer ved bruk av tydelige størrelsesforhold mellom elementer.
+
+### Nøkkelord
+
+-   Nivåer
```

Comment:
- Bruke nivåer
- Løfte viktig innhold
- Skape et tydelig hierarki

Kommentar: Når vi lister opp nøkkelordene, er det lurt å å starte på samme måte, for eksempel med et verb, slik du hadde gjort i punkt 2 med "løfte". Velg enten uten et verb på alle eller med.

Reply:
Enig i at det funker bedre! Disse er saksa rett fra profilguiden fra SDG 😅